src/main/java/mil/dds/anet/beans/Position.java

```diff
@@ -224,10 +226,30 @@ public Location getLocation() {
         });
   }
 
+  public CompletableFuture<List<PersonPositionHistory>> loadAllPeoplePositionHistory() {
```

Comment:
This method is only called by `getAllPeoplePositionHistory` below.

Reply:
This method is removed.